src/components/vm/disks/diskAdd.jsx

```diff
@@ -63,7 +63,22 @@ function getFilteredVolumes(vmStoragePool, disks) {
     return filteredVolumesSorted;
 }
 
-const SelectExistingVolume = ({ idPrefix, storagePoolName, existingVolumeName, onValueChanged, vmStoragePools, vmDisks }) => {
+function getDiskUsageMessage(vms, storagePool, volumeName) {
+    const isVolumeUsed = getStorageVolumesUsage(vms, storagePool);
+
+    if (!isVolumeUsed[volumeName] || (isVolumeUsed[volumeName].length === 0))
+        return null;
+
+    const vmsUsing = isVolumeUsed[volumeName].join(', ');
+    let text = _("This volume is already used by ") + vmsUsing;
+    const volume = storagePool.volumes.find(vol => vol.name === volumeName);
+    if (volume.format === "raw")
+        text += ". " + _("Attaching it will make this disk shareable for every VM using it.");
```

Comment:
This kind of addition may also be problematic. It's not a given that every language uses a period. Honestly, I'd rather compromise a little and always add the period to the "already used by ..." message. Otherwise you need *two* string templates which just differ by ".", and that's a little rude towards translators. But this doesn't map grammar-wise very well. E.g. in German you would always end a sentence with a period. (Honestly it's rather confusing that English doesn't? or is that just some PF guideline? If so, it's very English centric.)

Alternatively, force a line break and drop the period from both sentences? Then there's at least some consistency in the UI.

Reply:
I like the line break idea. We use something similar for snapshots:
![Uploading Screenshot from 2022-07-14 20-17-50.png…]()

Reply:
On the second thought, it seems that pattenrfly forces one-liner text for Form helperText by ignoring `\n`. We could do some workaround, but maybe it's just simpler to use sentences with dots on the same line.

Reply:
+1 on just using proper sentences, yes.

Reply:
Reverted back to using sentences ending with a comma. But let's wait if @garrett agrees.

Reply:
You didn't wait before merging.